File: ProjetoPadraoDotnetCore/Aplication/Models/Response/Chat/ContatoResponse.cs
﻿using Infraestrutura.Enum;

namespace Aplication.Models.Response.Chat;

public class ContatoResponse 
{
    public string? Nome { get; set; }
    public string? Foto { get; set; }
    public string? Telefone { get; set; }
    public string? Email { get; set; }
    public string? Sobre { get; set; }
    public string? DataNascimento { get; set; }
    public StatusContato StatusContato { get; set; }
    public int IdUsuarioContato { get; set; }
    public int? IdContatoChat { get; set; }
}

File: modern/src/app/objects/Chat/ContatoResponse.ts
import { EStatusContato } from "src/app/enums/EStatusContato";
import { RetornoPadrao } from "../RetornoPadrao";

export interface ContatoResponse extends RetornoPadrao
{
    data: DataContatoResponse
}

export interface DataContatoResponse
{
    nome: string,
    telefone: string,
    sobre: string,
    email: string,
    dataNascimento: string,
    foto: string,
    online: boolean,
    statusContato: EStatusContato,
    idUsuarioContato: number
    idContatoChat: number
}


